### PR TITLE
Fix gem publishing

### PR DESCRIPTION
**Pull request by fatbusinessman**
Our gem-publishing code stopped working today, as a result of (we can only assume) a change in the code on rubygems.org that stopped it accepting gems generated by older version of Ruby. This fixes the problem by pinning the Ruby version to 2.1 for the publishing step; there are also some fixes inspired by (read: shamelessly nicked from) the [equivalent changes in `gds-api-adapters`](https://github.com/alphagov/gds-api-adapters/pull/273).

### CODE REVIEW

File: jenkins.sh [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
+done
+
+unset RBENV_VERSION
 
-if [[ -n "$PUBLISH_GEM" ]]; then
-  bundle exec rake publish_gem --trace
-fi
+#if [[ -n "$PUBLISH_GEM" ]]; then
+  #bundle exec rake publish_gem --trace
+#fi
```
Comment (alext) [inline jenkins.sh]: I don't think you meant to leave this commented out...

Comment (fatbusinessman) [inline jenkins.sh]: I think you are entirely right :flushed: 

Amended in 6f565c2.

**Comment by dsingleton** [comment]
This looks good to me, any reason not to merge?

**Comment by fatbusinessman** [comment]
> This looks good to me, any reason not to merge?

None that I know of. With any luck, it’ll also get version 7.0.0 released and make @edds happy.

**Comment by fatbusinessman** [comment]
[Booyah](https://rubygems.org/gems/slimmer/versions/7.0.0).

**Pull request merged**